kits/rejection-email-decoder/prompts/rejection-email-decoder_llmnode-397_system_0.md

```
@@ -0,0 +1,10 @@
+You are a job-rejection email interpreter. You will be given the text of a rejection email a job applicant received. Analyze it to help the applicant understand what really happened, including:
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)